packages/web-client/app/models/workflow/milestone.ts

```diff
@@ -9,15 +9,19 @@ interface MilestoneOptions {
 export class Milestone {
   title: string;
   postables: WorkflowPostable[] = [];
-  includedPostables = new Set<WorkflowPostable>();
-  excludedPostables = new Set<WorkflowPostable>();
+  #includedPostables = new Set<WorkflowPostable>();
+  #excludedPostables = new Set<WorkflowPostable>();
```

Comment:
What does the # in the property name signify?

Reply:
Private class fields I think!

https://www.typescriptlang.org/play?ts=3.8.3#example/private-class-fields

Reply:
Ah is it just to mean number? At first I thought it was some new ember or typescript thing lol.

Reply:
Ok so it was for typescript. I've only seen _ used before.

Reply:
"Private"! It's a newish JS feature now in Stage 3: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
"_" is just a convention that communicates "this is private". "#" is part of Javascript that enforces privacy.